app/models/project_attributes.rb

```diff
@@ -0,0 +1,8 @@
+class ProjectAttributes < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :project
+
+  def project
+    Project.find(self.project_id)
+  end
```

Comment:
We override it since ActiveRecord method does not behaviour as we expect
(this was a copy of ProjectOwnership).

Em qui, 26 de fev de 2015 às 13:44, Diego de Araújo Martinez Camarinha <
notifications@github.com> escreveu:

> In app/models/project_attributes.rb
> <https://github.com/mezuro/prezento/pull/183#discussion_r25442242>:
>
> > @@ -0,0 +1,8 @@
> > +class ProjectAttributes < ActiveRecord::Base
> > +  belongs_to :user
> > +  belongs_to :project
> > +
> > +  def project
> > +    Project.find(self.project_id)
> > +  end
>
> ActiveRecord already gives us this method, doesn't it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mezuro/prezento/pull/183/files#r25442242>.
>